participants/SwiftCloudBackup.md

```diff
@@ -0,0 +1,71 @@
+# Swift Cloud Backup System
+
+## (0) Project Name and Team Slack Handles
+
+*Please tell us your project's name and the Filecoin Slack handles for your teammates.*
+
+Project name: **SwiftCloudBackup**
+Filecoin Slack handle: **coder-lb**
+
+## (1) Brief Project Description
+
+Enterprises generate large amounts of data every day. Valuable data have become important assets of enterprises. In order to make these data available and security, reliable data backup services in remote locations must be used. StorSwift is an enterprise data storage provider and helps many enterprises to design their backup system. traditional backup methods have many defects, i.e., costs, privacy, availability, and so on.   
+So we are trying to find better solution for enterprise cloud backup system. Filecoin is a brand new storage technology based on zk-SNARKs and block chain, and we think it can give enterprises more choices. It may reduces the costs because of sunk costs. It can build a better solution for privacy since it is decentralized. It will improve the performance and availability because people can choose the best miner nodes if there are thousands of miners continue to provide storage & retrieval services in the network. So we develop and design an enterprise cloud backup system based on the Filecoin network, and believe it will be one of our most important storage businesses in the future.
+
+## (2) Link to Project UI
+
+https://cloudbackup.storswift.com/
+
+It is under development, and we already have an internal web application for testing. It will be online very soon!
+
+## (3) What does your application/UI do?
+
+
+People can upload their archive files to the specific miner nodes via this application. They can view their files with the corresponding CIDs and download them easily. It can also record the backup history.  
+
+## (4) If your project is using a curated dataset, which one are you using?
+
+Our project use the archive data of enterprises or non-profit organizations. Of course we can use the curated datasets to test, but we prefer to the archive datasets from our partners.
+
+## (5) If your project is not using a curated dataset, please tell us a little bit more about your data by answering the questions below.
+
+We will use the archive data with public license for this project. These data include private and public ones. Private data can only be accessed by the owners. Some of data are public, and we ask each user to add the corresponding license in the archive package (a text file in the root folder before compressing). We also have provided some public docker & VM images to many customers, and they are public data but very useful for enterprise IT managers.
```

Comment:
For this competition, we will require to retrieve and  validate the data stored on the network.  Please consider in your design.